### PR TITLE
fix: added missing openFile method in ios utils

### PR DESCRIPTION
**Pull request by vakrilov**
Export `openFile` method for in IOS utils.
Related to #5661

### CODE REVIEW

File: tns-core-modules/utils/utils.ios.ts [hunk 1]
```diff
@@ -146,6 +146,8 @@ export module ios {
 
 }
 
+export const openFile = ios.openFile;
```
Comment (manoldonev) [inline tns-core-modules/utils/utils.ios.ts]: Maybe we should deprecate ios.openFile then?

Comment (vakrilov) [reply tns-core-modules/utils/utils.ios.ts]: Moved implementation to the root and deprecated the `utils.ios.openFile()` method

**Comment by SvetoslavTsenov** [comment]
test

**Comment by vakrilov** [comment]
@cla-bot check

**Pull request merged**